app/public/css/styles.css

```diff
@@ -3207,14 +3207,24 @@ button[data-balloon] {
   }
 
   /* TOOLTIP */
+  
+  [data-balloon]::after {
+    position: fixed;
+    top: 90%;
+    left: 50%;
+    margin-left: -77px;
+    bottom: unset;
+  }
 
-  [data-balloon]:hover::after,
-  [data-balloon]:hover::before,
-  [data-balloon]::after,
   [data-balloon]::before {
     display: none;
   }
 
+  [data-balloon]::hover::after,
+  [data-ballon]::hover::before{
+    display: none;
+  }
```

Comment:
these two can be joined together if they do same stuff

Reply:
fixed